github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits-v0.31.yaml

```diff
@@ -0,0 +1,66 @@
+---
+presubmits:
+  k8snetworkplumbingwg/kubemacpool:
+    - name: pull-kubemacpool-unit-test-v0.31
+      branches:
+        - release-v0.31
+      annotations:
+        fork-per-release: "true"
+      always_run: true
+      optional: false
+      decorate: true
+      decoration_config:
+        timeout: 3h
+        grace_period: 5m
+      cluster: phx-prow
+      max_concurrency: 6
```

Comment:
maybe better to reduce it to 2 ?
same on the 2nd job

Reply:
we can manage with 4, will reduce to avoid taking too much resources from ci.